pages/activities/ActivitiesPage.jsx

```diff
@@ -100,13 +100,12 @@ var ActivitiesPage = React.createClass({
               developedBy={<div><a href="http://en.mozillafactory.org/tagged/COMOZILLA">Mozilla Factory</a> and <a href="http://twitter.com/smithisgeneric">Karen Smith</a> and remixed by <Link to={"/"}>Mozilla Learning Networks</Link>.</div>}
               description="Learn to use Mozilla Factory’s Parapara and Fabble, open web tools for animation and sharing work online. Create 2D animations, 3D projection-mapped animations, and ‘recipes’ that others can fork for remix on Fabble."/>
             <ActivityKit
-              src1x="/img/pages/activities/img-activity-03.png"
-              src2x="/img/pages/activities/img-activity-03@2x.png"
```

Comment:
Since we are not using these images anymore. We should remove them from the repo.

1. Check to see if the image file is being referenced somewhere else in the codebase. Your text editor should have the option for you to search within an entire folder. You can use the file name as the search term, e.g.,`img-activity-03.png`
2. If the image file isn't being used elsewhere, remove it from repo. For example,
```
git rm img/pages/activities/img-activity-03.png
```